gradle/libs.versions.toml

```diff
@@ -84,6 +84,7 @@ spark35 = "3.5.5"
 sqlite-jdbc = "3.49.1.0"
 testcontainers = "1.21.0"
 tez08 = { strictly = "0.8.4"}  # see rich version usage explanation above
+comet = "0.8.1"
```

Comment:
nit: please move this further up (right after `calcite`)

Reply:
Fixed. Thanks!